samples/core/component_build/component_build.ipynb

```diff
@@ -88,178 +81,79 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!gsutil mb {OUTPUT_DIR}"
+    "def add(a: float, b: float) -> float:\n",
+    "    '''Calculates sum of two arguments'''\n",
+    "    \n",
+    "    print(\"Adding two values %s and %s\" %(a, b))\n",
+    "    \n",
+    "    return a + b"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Create an Experiment in the Pipeline System\n",
-    "\n",
-    "Pipeline system requires an \"Experiment\" to group pipeline runs. You can create a new experiment, or call client.list_experiments() to get existing ones."
+    " ## Build a new container for and use is as the base image for components.\n",
+    " Build and push a new container image that includes some required packages. The packages are specified in the requirements.txt file in the working directory.\n",
+    " Use the new container image as the base image for components.\n",
+    " Create a new component from the above function.\n",
+    " The return value \"add_op\" represents a step that can be used directly in a pipeline function."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "#Get or create an experiment and submit a pipeline run\n",
-    "import kfp\n",
-    "client = kfp.Client()\n",
-    "\n",
-    "try:\n",
-    "    experiment = client.get_experiment(experiment_name=experiment_name)\n",
-    "except:\n",
-    "    experiment = client.create_experiment(experiment_name)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Create a python function"
+    "from pathlib import Path\n",
+    "print(Path('requirements.txt').read_text())"
```

Comment:
Would it make sense to have this as mark up rather than code? execution of this line will case an error in the notebook as the env may not have the requirements.txt already created in it. 

Alternatively you could create the requiremetns.txt first.

Reply:
The commit that introduced this line includes the `requirements.txt` which resides in the same directory as the notebook.